pkg/apis/core/validation/validation.go

```diff
@@ -3720,10 +3758,71 @@ func ValidatePodCreate(pod *core.Pod, opts PodValidationOptions) field.ErrorList
 	if len(pod.Spec.EphemeralContainers) > 0 {
 		allErrs = append(allErrs, field.Forbidden(fldPath.Child("ephemeralContainers"), "cannot be set on create"))
 	}
+	allErrs = append(allErrs, validateSeccompAnnotationsAndFields(pod.ObjectMeta, &pod.Spec, field.NewPath("spec"))...)
```

Comment:
I'm not sure if this is the only place, but there might be the case that we end-up having having an error list of `[<nil>]` here. When I'm trying out the PR https://github.com/kubernetes/kubernetes/pull/91408, then the issue is already reproducible when running hack/local-up-cluster.sh (see the kubedns replicaset).

Reply:
Thank you for pointing it out, I have just pushed a fix for it.